### ISSUE TITLE
Dash style unordered list seen as asterisk inside block quote

### ISSUE THREAD

**Issue by brandonrisell** [DavidAnson/markdownlint#104]
When you have a dash stye unordered list inside of a block quote, the markdownlint VSCode extension tells you that it expected a dash, but the actual is an asterisk. If I remove the block quote character it is fine with the dash.

E.g.
```markdown
>- Item 1
>- Item 2
>- Item 3
```

I am using the vscode extension `markdownlint` version `0.13.0`, and the rule that is triggering, `MD004`, is using it's default value.

**Comment by DavidAnson** [comment]
When pasted into https://dlaa.me/markdownlint/ your sample above does not trigger that error. Is there something else that’s necessary to make it happen?

**Comment by brandonrisell** [comment]
So this took a bit to sort out, because you're right, I copied in the example and then even parts of the document and couldn't get the error message.

I finally figured it out though. It's if there was an unordered list earlier in the document. This is super simple, but triggers the error message I'm seeing.

```markdown
# Header 1

- Item 1

>- List item
>- List item
```

In our case there are a bunch of things between the normal unordered list and the later quoted unordered list, but that doesn't seem to impact it.

**Comment by DavidAnson** [comment]
Not looking at the code, but have an idea what’s going on. Thanks for the example!
